modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/internal/ConnectionClientPool.java

```diff
@@ -670,8 +679,11 @@ public boolean removeNodeClient(UUID nodeId, GridCommunicationClient rmvClient)
         for (; ; ) {
             GridCommunicationClient[] curClients = clients.get(nodeId);
 
-            if (curClients == null || rmvClient.connectionIndex() >= curClients.length || curClients[rmvClient.connectionIndex()] != rmvClient)
+            if (curClients == null
+                || rmvClient.connectionIndex() >= curClients.length
+                || curClients[rmvClient.connectionIndex()] != rmvClient) {
```

Comment:
The next statement is one-liner. We don't need braces here.

Reply:
Fixed